Cargo.toml

```diff
@@ -13,8 +13,12 @@ dotenv_codegen = "0.15.0"
 regex = "1"
 lazy_static = "1.4.0"
 
-rocket = { git = "https://github.com/SergioBenitez/Rocket", rev = "4f3511786cb8b6540124b867591c7c5a6933409e" }
-rocket_codegen = { git = "https://github.com/SergioBenitez/Rocket", rev = "4f3511786cb8b6540124b867591c7c5a6933409e" }
+rocket = { git = "https://github.com/SergioBenitez/Rocket", rev = "dd9629697a3cbe45f50645579f41f1a957a92968", features = ["tls", "json"] }
+rocket_codegen = { git = "https://github.com/SergioBenitez/Rocket", rev = "dd9629697a3cbe45f50645579f41f1a957a92968" }
+
+# Force rocket peer-dependencies versions
+proc-macro2 = "1.0.27"
+tokio = "1.6.1"
```

Comment:
Is this actually okay? I think `rocket` exposes the version of async that it uses, if we force a different version onto it we could have issues.

Reply:
Also curious about why we need `proc-macro2` as well. Same issue?

Reply:
This corresponds to the version that `rocket` requires. If the versions are not compatible the compiler will complain